go_pprof_scraper/datadog_checks/go_pprof_scraper/__about__.py

```diff
@@ -0,0 +1,4 @@
+# (C) Datadog, Inc. 2022-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+__version__ = '0.0.1'
```

Comment:
Do you want to keep 0.0.1 or have 1.0.0 directly once merged?

Reply:
I doubt we'll need to change this much once it's merged, so I'll update this to 1.0.0.